### DISCUSSION TITLE
Are there any default template repos?

### DISCUSSION THREAD

**Discussion by nmaludy** [puppetlabs-toy-chest/pct#109] [Feedback]
Scanning the README and i'm pretty excited about PCT. Curious if there are any existing template repos we can look at to gain some insight?

Thanks!

**Comment by jpogran** [comment]
The tool comes with a set of experimental templates that are stored in https://github.com/puppetlabs/baker-round. These are purposely stored in a temporary location while we get feedback. 

The intention is to graduate these to repos of their own when we get support for hosting the templates on the Puppet Forge. We intend to have a Puppet module type contribution experience, where some are published under Puppetlabs but others by the community, so that there is choice for the user to pick what works best for them.
